plugins/linux/l3plugin/route_config.go

```diff
@@ -471,6 +494,19 @@ func (plugin *LinuxRouteConfigurator) ResolveDeletedInterface(name string, index
 	return nil
 }
 
+// RouteIdentifier generates unique route ID used in mapping
+func (plugin *LinuxRouteConfigurator) RouteIdentifier(route *netlink.Route) string {
+	var dstAddr string
```

Comment:
You could use [net.IPNet](https://godoc.org/net#IPNet) for the purpose you are using here, which would eliminate the need to do so many checks and string concatenating.